internal/persistence/jsondb/database.go

```diff
@@ -307,9 +309,14 @@ func (store *Store) pattern(dagFile string) string {
 	return filepath.Join(dir, p)
 }
 
-func (store *Store) latestToday(dagFile string, day time.Time) (string, error) {
```

Comment:
I think we can use the following method instead of modifying this one:

https://github.com/dagu-dev/dagu/blob/151d842411e2492dd5e52f8852833e524fe39e7c/internal/persistence/jsondb/database.go#L139-L140

We can modify the following code to use the `ReadStatusRecent` method according to the configuration:

https://github.com/dagu-dev/dagu/blob/151d842411e2492dd5e52f8852833e524fe39e7c/internal/engine/engine.go#L189-L203